### ISSUE TITLE
WCAG 2.2 content update: Tag || Target size (minimum)

### ISSUE THREAD

**Issue by davidc-gds** [alphagov/govuk-design-system#2656]
## What
Check whether the target area of tags needs to be adjusted so that the target area has a height of at least 24px.

Note: the typography work may affect this, as it is changing how font size is handled for small, mobile-size screens.

### Resources
- [Tag component](https://design-system.service.gov.uk/components/tag/)
- [WCAG 2.2 criterion](https://www.w3.org/TR/WCAG22/#target-size-minimum)

## Why
Currently tags on small, mobile-size viewport widths are rendered with a target area height of 23px. On large, desktop and tablet-size viewport widths, the height of the target area is 25px.

It would be better if we just bump small-screen up to be at least 24px height.

The relevant part of the criteria is:

> "The size of the target for pointer inputs is at least 24 by 24 CSS pixels..."

Technically our current tag should be fine from a 'passes WCAG 2.2 AA' sense, in 99% of situations. However, the current form could fail AA on mobile-size screen widths, if a bunch of tags are stacked vertically without spacing between them.

## Who needs to work on this
Developer
## Who needs to review this
Accessibility specialist

## Done when

- [ ] Investigate the issue
- [ ] Determine whether the typography changes will fix this by bumping up the size of our mobile fonts
- [ ] Decide whether we see any value in adding a minimum width / height for the tag target area
- [ ] Make the change
- [ ] Review the change (accessibility specialist)
- [ ] Merge

**Comment by querkmachine** [comment]
Does this still apply if the tags aren't links?

We don't _officially_ support using tags as links, or having links inserted into the tag text, so the component (as we ship it) could be considered compliant already. 

However, we know that using them as links [is something that folks do sometimes](https://github.com/alphagov/govuk-design-system-backlog/issues/62#issuecomment-885615572), so it might be worth making that functionality official, or increasing the tag's height preemptive to services using them as links.

**Comment by davidc-gds** [comment]
Hey @querkmachine! Ahh yes, you've noted the other assumption made. Yes, this is more of a better-safe-than-inaccessible thing, in the event that a service team chooses (against our guidance) to make a tag into a link 👍 

I'll add that tidbit to the issue description!

**Comment by owenatgov** [comment]
Had a light chat about this with @querkmachine and we've agreed that we'll keep an eye on this as part of https://github.com/alphagov/govuk-design-system/issues/2299 and see if there's an opportunity to tie this into the review of the tag component specifically.

**Comment by querkmachine** [comment]
Some of the proposed designs in #3502 also increase the height of the tag component, also through increasing the text size, so this issue may be resolved as part of that work too.

**Comment by owenatgov** [comment]
Just hopping in to add that from auditing the components against the new typography scale as part of https://github.com/alphagov/govuk-design-system/issues/2300, I can confirm that this will be solved as part of that update.

**Comment by davidc-gds** [comment]
> Just hopping in to add that from auditing the components against the new typography scale as part of #2299, I can confirm that this will be solved as part of that update.

Ah excellent! I thought that might be the case. 

Sounds like this specific one will either be solved by the tag updates from the task-list working group, or from the new typography scale. A race to the finish 🏃
